packages/stacking/tests/stacking.test.ts

```diff
@@ -115,7 +116,7 @@ test('check stacking eligibility false bad cycles', async () => {
   const errorType = new BN(2);
   const expectedErrorString = StackingErrors[StackingErrors.ERR_STACKING_INVALID_LOCK_PERIOD];
   // CV responseError value = CV uint 2
-  const functionCallResponse = { type: 8, value: { type: 0, value: errorType } } 
+  const functionCallResponse = { type: 8, value: { type: 0, value: errorType } }
```

Comment:
Can you replace this with:

```typescript
const functionCallResponse = responseErrorCV(uintCV(2));
```

Reply:
Looking at the error types, an `int` is required (https://docs.blockstack.org/references/stacking-contract#err_stacking_invalid_lock_period). This worked:

```
const functionCallResponse = responseErrorCV(intCV(2));
```